src/utils/resources/udns/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { modelToGroupVersionKind, modelToRef } from '@kubevirt-ui/kubevirt-api/console';
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+export const UserDefinedNetworkModel: K8sModel = {
+  abbr: 'UDN',
+  apiGroup: 'k8s.ovn.org',
+  apiVersion: 'v1',
+  crd: true,
+  id: 'userdefinednetwork',
+  kind: 'UserDefinedNetwork',
+  label: 'User Defined Network',
+  labelPlural: 'User Defined Networks',
+  legacyPluralURL: true,
+  namespaced: true,
+  plural: 'userdefinednetworks',
+};
```

Comment:
In my pr i created this exact model into `src/utils/models/index.ts`
For me it's okay to have it here but let's just be aware that when both prs will be merged we don't want to different UDN model definition

Reply:
yeah I saw that after creating this PR :smile_cat: 

I will point your definition as soon as your PR is merged